src/i18n.ts

```diff
@@ -171,16 +178,11 @@ should either enable sessions or use `ctx.i18n.useLocale()` instead.",
       }
 
       // deno-lint-ignore no-explicit-any
-      (await (ctx as any).session).__language_code = locale;
+      (await (ctx as any).session).__language_code = localeId;
+      locale = localeId;
```

Comment:
I'm not sure that this is where locale should be updated. I'd rather move it to useLocale (i.e. where it is actually used in Fluent)

Reply:
Or even maybe make it a proxy/shortcut? `ctx.i18n.locale === ctx.i18n.fluent.something`. I think that the current locale should be available in fluent somewhere

Reply:
> Or even maybe make it a proxy/shortcut? `ctx.i18n.locale === ctx.i18n.fluent.something`. I think that the current locale should be available in fluent somewhere

Even though yours was a very good idea, we actually don't have any way to get locale id directly from fluent object. Right now, all we have are functions for configure Fluent setup :? No public property was found. The way we work with locale id in fluent is actually bind it to fluent translate function. The fluent instance simply doesn't keep local id value.

Reply:
> I'm not sure that this is where locale should be updated. I'd rather move it to useLocale (i.e. where it is actually used in Fluent)

I'm aware about this problem as well and try to fix it. Thank for giving a good confirmation.